docs/plugins/structure-of-a-plugin.md

```diff
@@ -57,9 +57,10 @@ In the `src` folder we get to the interesting bits. Check out the `plugin.ts`:
 
 ```jsx
 import {
-  createPlugin,
-  createRoutableExtension,
```

Comment:
This is all old documentation too, so they should be using the `core-plugin-api` packages, not sure we should be surfacing the new `frontend-plugin-api` packages here.